parlai/mturk/core/mturk_data_handler.py

```diff
@@ -669,3 +727,39 @@ def get_all_task_pairings_for_worker(self, worker_id, task_group_id=None):
                       (worker_id, task_group_id))
             results = c.fetchall()
             return results
+
+    @staticmethod
+    def get_conversation_data(task_group_id, conv_id, worker_id, is_sandbox):
```

Comment:
for some reason, this way of returning the data seems very satisfying to me

Reply:
Once the data is in an organized place, it's really easy to access it. The accessors along the way make sure that if someone manually alters the folder structure there is no catastrophic failure and I can give a good reason.